content/reference/service-map.adoc

```diff
@@ -1,12 +1,130 @@
 = Service Map
-TBD
-2016-08-22
+Michael Glaesemann
+2016-09-03
 :jbake-type: page
 :toc: macro
 :icons: font
 :section: reference
 
+== Description
 
-== Placeholder
+The Service Map is a map that defines the logic of the service that will
+be created when the server starts.
 
-**Pull requests welcomed!**
+[cols="s,d,d,d", options="header", grid="rows"]
+|===
+| Key | Always present? | Type | Description
+
+| `:io.pedestal.http/routes`
+| Y
+| Sequence of maps, function, ExpandableRoutes
```

Comment:
This is a sequence of maps, a single function, or a sequence of ExpandableRoutes, right? Current wording makes it seem like all are sequences. (The description clarifies, but we can be clear in the type as well.)